graphql/src/schema/api.rs

```diff
@@ -212,19 +212,39 @@ fn field_scalar_filter_input_values(
     field: &Field,
     field_type: &ScalarType,
 ) -> Vec<InputValue> {
-    vec![
-        Some(input_value(
-            &field.name,
+    match field_type.name.as_ref() {
+        "BigInt" => vec!["", "not", "gt", "lt", "gte", "lte", "in", "not_in"],
+        "Boolean" => vec!["", "not", "in", "not_in"],
+        "Bytes" => vec!["", "not", "in", "not_in", "contains", "not_contains"],
+        "Float" => vec!["", "not", "gt", "lt", "gte", "lte", "in", "not_in"],
+        "ID" => vec!["", "not", "gt", "lt", "gte", "lte", "in", "not_in"],
+        "Int" => vec!["", "not", "gt", "lt", "gte", "lte", "in", "not_in"],
+        "List" => vec!["", "not", "in", "not_in", "contains", "not_contains"],
+        "String" => vec![
             "",
-            Type::NamedType(field_type.name.to_owned()),
-        )),
-        Some(input_value(
-            &field.name,
             "not",
-            Type::NamedType(field_type.name.to_owned()),
-        )),
-    ].into_iter()
-        .filter_map(|value_opt| value_opt)
+            "gt",
+            "lt",
+            "gte",
+            "lte",
+            "in",
+            "not_in",
+            "contains",
+            "not_contains",
+            "starts_with",
+            "not_starts_with",
+            "ends_with",
+            "not_ends_with",
+        ],
+        _ => vec!["", "not"],
+    }.into_iter()
+        .map(|filter_type| {
+            input_value(
+                &field.name,
+                filter_type,
+                Type::NamedType(field_type.name.to_owned()),
```

Comment:
It seems that for `in` and `not_in` the correct type here would be `[Type]` (a list of `Type`) rather than just `Type`, so that in the docs we get for example `deposit_in: [BigInt]` rather than `deposit_in: BigInt` which is what currently appears.